### DISCUSSION TITLE
Any better ideas / features for phone-formatter-eth npm package?

### DISCUSSION THREAD

**Discussion by HuluWZ** [HuluWZ/phone-formatter-eth#5] [Ideas]
Any better ideas / features for phone-formatter-eth npm package?

**Comment by eyopa21** [comment]
1. parse - Cleans up or normalizes phone.e.g special characters like - and ()
2. toLocal - Converts phone number to local | Ethiopian format. 09 or 07 format
3. toInternational - Converts phone number to Int'l | Ethiopian format. +2519/7
4. isMobile - check if it's mobile sim. eg 09/07/+2519/7
5. isLandline - check if it's landline sim. eg. +2511
